cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/lineage/field/DefaultFieldLineageReader.java

```diff
@@ -118,14 +117,11 @@ public DefaultFieldLineageReader(DatasetFramework datasetFramework, TransactionS
         FieldLineageInfo info = new FieldLineageInfo(programRunOperation.getOperations());
         Set<Operation> fieldOperations = incoming ?
                 info.getIncomingOperationsForField(new EndPointField(endPoint, field))
-                : info.getOutgoingOperationsFromField(new EndPointField(endPoint, field));
+                : Collections.EMPTY_SET;
```

Comment:
Returning EMPTY_SET for outgoing operations since the work is not planned for 5.0. Will add implementation later if required.